docs/localization/README.md

```diff
@@ -0,0 +1,10 @@
+# CNCF Localization Guides 
+
+This directory contains a growing collection of localization guides that CNCF projects can freely use, copy, and adapt.
+
+Every guide here must meet the following requirements:
```

Comment:
I like that you're setting requirements expectations here -- we may want to adopt this more generally.